students/rodicaf/tests/fibonacci.test.ts

```diff
@@ -0,0 +1,10 @@
+import {fibrec, fibit} from '../src/fibonacci';
+
+describe("Given a fibrec function", () => {
```

Comment:
missing more case test. Additionally test description should be written using BDD (Given, When Then)